### PR TITLE
Added a vendor sales page

### PR DESCRIPTION
**Pull request by ritwizsinha**
image

### CODE REVIEW

**Comment by ritwizsinha** [comment]
Screenshot as attached above
Did not hardcode the html and hardcoded only the state to make it easier to change once actual data comes

**Comment by harsh-jindal** [comment]
@ritwizsinha The issue was to create a page where the vendor can enter his sale/business information when vendor is logged in.
We will also have a page that will be accessible by everyone. But the UI would be kept more similar to the justdial page.
So either make the UI similar to just dial or add input fields where vendor can enter his details.

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -104,7 +104,11 @@ class Navbar extends Component {
               <Switch>
                 <Route exact path="/Profile" component={Profile} />
                 <Route exact path="/userDashboard" component={UserDashboard} />
-                <Route exact path="/vendorDashboard" component={VendorDashboard} />
+                <Route
```
Comment (harsh-jindal) [inline src/components/Navbar/Navbar.js:107]: Keep this in single line. Breaking up each route in separate line simply makes the code long and difficult to read in future.

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (ritwizsinha) [reply src/components/Navbar/Navbar.js:107]: Changes made , it was the editors fault which reset it to this

**Comment by ritwizsinha** [comment]
image
These ar the corresponding input fields

**Review by harsh-jindal — APPROVED** [review]

**Comment by harsh-jindal** [comment]
LGTM 👍 . Few spelling mistakes which can be fixed later on.

**Pull request merged**